amundsen_application/static/js/components/common/ResourceListItem/TableListItem/tests/index.spec.tsx

```diff
@@ -126,26 +127,36 @@ describe('TableListItem', () => {
         expect(resourceBadges.exists()).toBe(true);
       });
 
-      describe('if props.table has last_updated_timestamp', () => {
-        it('renders Last Updated title', () => {
-          expect(resourceBadges.children().at(0).children().at(0).text()).toEqual('Last Updated');
+      describe('if props.table has badges', () => {
+      
+        it('renders flag for badge', () => {
```

Comment:
nit: `'renders flag for badge'` --> `'renders BadgeList for badges'`

not a nit: For a better test we want to check not just that a component exists but verify that `TableListItem` gave it the right properties. So we could update this text to be:
```
expect(resourceBadges.find(BadgeList).props().badges).toEqual(props.table.badges);
```
That one assertion will end up failing if `BadgeList` doesn't exist but also if it doesn't get the right properties passed, making it a more useful test.

Reply:
Super useful, thanks so much! I was trying to figure out how to test that the correct values were present. I hadn't thought of trying the props.